### DISCUSSION TITLE
Is OpenGL going to released soon for Severed Chains? Or is it planned for 2024 some time?

### DISCUSSION THREAD

**Discussion by linlinxza** [Legend-of-Dragoon-Modding/Severed-Chains#960] [General]
Is OpenGL going to released soon for Severed Chains? Or is it planned for 2024 some time? Is there a way to enable it if it's available in the dev builds?

**Comment by LordMonoxide** [comment]
Hi @linlinxza, we are aiming to have it finished in the first quarter of 2024, and we will most likely have public test builds out in less than a month. We also have private test builds available now. If you're interested in joining the private test, ping me on the Discord server and I'll get you set up.

**Reply by linlinxza** [reply]
I prefer to just try out the private test builds and test out the experimental version of it with OpenGL. But I refuse to go on Discord. I hate it with a passion.

**Comment by linlinxza** [comment]
> Hi @linlinxza, we are aiming to have it finished in the first quarter of 2024, and we will most likely have public test builds out in less than a month. We also have private test builds available now. If you're interested in joining the private test, ping me on the Discord server and I'll get you set up.

So, can I have access to the private build?
